Sources/Commands/SwiftTool.swift

```diff
@@ -117,11 +117,11 @@ private class ToolWorkspaceDelegate: WorkspaceDelegate {
     }
 
     func willUpdateRepository(package: PackageIdentity, repository url: String) {
-        self.outputHandler("Updating \(url)", false)
+        self.outputHandler("Fetching updates from \(url)", false)
```

Comment:
Maybe "Checking for updates from..." @neonichu @abertelrud opinions?

Reply:
This wording sounds good to me since we don't know whether there will be any.  Ideally we should later add another callback for if there actually are any, but that's a later PR.